ui/app/serializers/config-ui/message.js

```diff
@@ -3,15 +3,37 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+import { encodeString } from 'core/utils/b64';
 import ApplicationSerializer from '../application';
 
 export default class MessageSerializer extends ApplicationSerializer {
   primaryKey = 'id';
 
+  serialize() {
+    const json = super.serialize(...arguments);
+    json.message = encodeString(json.message);
```

Comment:
For custom messages, we discussed encoding the string when it's sent to the backend and decoding it when we show the message on the details screen as well as on the authenticated / unauth screens.